src/domUpdates.js

```diff
@@ -6,17 +6,25 @@ let domUpdates = {
     let bookings = currentCustomer.returnBookings(type, currentDate)
     let bidetMessage;
     bookings.forEach(booking => {
-      booking.bidet ? bidetMessage = 'Hooray there\'s a bidet!' : bidetMessage = 'No bidet for you'
+      booking.bidet ? bidetMessage = 'Luxury Bidet' : bidetMessage = 'Sorry, no bidet'
       element.innerHTML +=
       `
         <article class="booking-card">
-          <h5>${booking.date}</h4>
-          <p>${booking.roomType}</p>
-          <p>Room Number ${booking.roomNumber}</p>
-          <p>Beds: ${booking.numBeds} ${booking.bedSize}<p>
-          <p>$${booking.costPerNight}/ night</p>
-          <p>Bidet included?</p>
-          <p>${bidetMessage}</p>
+          <div class="card-header">
```

Comment:
I couldn't tab through your cards. As Ash said, look into "tabindex=0".

Reply:
yes good idea thank you!